### PR TITLE
Features

### PR DESCRIPTION
**Pull request by ros-a**
Hi Rayna!
If you could have a look whenever you have time that would be great.
All the images are placeholders so don't pay attention to that.
I'm also still going to add titles etc. to the projects. 
Cheers!

### CODE REVIEW

**Comment by norayna** [comment]
portfolio items only have a picture - it would be ideal to have a name and description and maybe even github link for each one, because you don't have a separate page for individual projects. i'd say to add:
- name of each project - for sure
- description - if possible
- github link - if possible

**Comment by norayna** [comment]
on mobile - portfolio images are slightly off to one side:
image
i assume they are supposed to be centered?

File: index.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+                <a href="#" class="projects project3">
+                    <div class="project-images project-image3"></div>
+                </a>
+                <a href="#" class="projects project4">
+                    <div class="project-images project-image4"></div>
+                </a>
+                <a href="#" class="projects project5">
+                    <div class="project-images project-image4"></div>
+                </a>
+                <a href="#" class="projects project6">
```
Comment (norayna) [inline index.html]: as far as i can tell the classes project1 to project6 and project-image1 to project-image6 are not doing anything? if so, you can remove them. also if there are any other classes that are not used for anything they should be removed as well - we create css classes only when they are needed

if they are doing something - bare in mind the class on the 5th project image is project-image4 - probably a mistake

**Review by norayna — CHANGES_REQUESTED** [review]
just the few comments there already

**Comment by ros-a** [comment]
Hi! I changed the bits you commented on. I also added titles to the projects, but as soon as I have any projects, I'd like them to open to a new page, where I'll then have a description and Github link :) so for now it's just the photo with a title but that will change as soon as I've finished my first project to put on there!

**Review by norayna — APPROVED** [review]
approved 🧅

**Pull request merged**